### ISSUE TITLE
LuckPerms forgets users

### ISSUE THREAD

**Issue by AaronFrost99** [lucko/LuckPerms#2883]
### Reproduction steps

1. User registered/login at server
2. User logout
3. I can see the user at web luckperms console
4. user missing for about a day
5. user info lost from web luckperms console

### Expected behaviour

It shouldn't be like this, or is there something I don't know?. 

### Environment details

git-Yatopia-13 (MC: 1.16.5)
* ,  Luckperms v5.2.104 

### Any other relevant details

These events were detected after the server was moved from sqlite storage to mysql. However, this may be a coincidence. However, the translation was also not quite correct - not all the data was transferred. Some of the data was not transferred. We don't know why. That is, when switching between storage types, the data is different. In the local format, Luckperms sees more information

**Comment by benwoo1110** [comment]
Note that there isn't any syncing of data between storage types. You need to export and then import the data. See: https://luckperms.net/wiki/Switching-storage-types#how-to

**Comment by AaronFrost99** [comment]
The data was exported and imported as the developer specified-with the appropriate /export and /import commands. The server reported a successful import of the database, and the data in the Mysql database was reflected, but not all.

**Comment by lucko** [comment]
It's unlikely that there's a bug that is causing this issue, perhaps you made a mistake during the migration?

If you still have the import/export files to hand, I can take a look to see if the data is there? You could always run the import again and it'll merge the old data back in again.

Closing for now, but please feel free to re-open if you have any more info to give.

**Comment by AaronFrost99** [comment]
Morgning My LuckPerms data was in Mysql type
I created the character "Test" and went into the game with it. After then relogin as an administrative character. In the webconsole, character was, i checked it. I applied the permission for the group (adding rtp for default), and then opened the lacperms console again - the "Test" character was no longer there - it just disappeared. 
There were no storage migrations between it 2 action. LuckPerms was and remained on Mysql. After that, I moved the storage from sql to H2 local, since as far as I understand, LuckPerms cannot work correctly with databases. If such an error occurs again in local storage, I will report it here.
However, given that there were no migrations between the registration of the character and its disappearance , no mistakes could be made in the migration process (since there was no migration itself).
